src/sentry/organizations/services/organization/impl.py

```diff
@@ -751,6 +751,12 @@ def send_signal(
     ) -> None:
         signal.signal.send_robust(None, organization_id=organization_id, **args)
 
+    def get_organization_members(self, *, organization_id: int) -> list[RpcOrganizationMember]:
+        org: Organization = Organization.objects.get(id=organization_id)
+        org_members = org.member_set.all()
+
+        return list(map(serialize_member, org_members))
```

Comment:
Some organizations have *many* members and the response time of this RPC could hit timeouts for larger customers as results need to be fetched, serialized and sent across the wire.

If you're in controlsilo, you could use `OrganizationMemberMap` to find members for an organization and save yourself some RPC overhead.

Reply:
i forgot that OrgMemberMapping has the ids and that's all i need. thank you!